### DISCUSSION TITLE
Page feedback feature improvements?

### DISCUSSION THREAD

**Discussion by chalin** [google/docsy#1805] [General]
From https://github.com/google/docsy/discussions/1719#discussioncomment-8319155 @jon-hatfield-tech-writing wrote:

> How about a separate event for 'page_unhelpful'? Then it would be possible to create a free form report (Explore > Free form) with a row of 'page title', column of 'event name' and value of 'event count' (you may need to add these properties as dimensions and metrics first on the left hand side). This would give a view of helpful / unhelpful clicks for each page.
> 
> In terms of the current implementation it's possible to add an 'event value' column to the table in the 'pages and screens' report then filter the 'event count' column to 'page_helpful'. The sum of helpful (100) and unhelpful (0) can then be seen for each page (or change the first column to 'event name' to get a summary). This view is not especially intuitive though so an export to CSV ('Share this report') may be necessary in order to then calculate the percentage of positive responses.

Thanks, I'll comment further later, but I wanted to create this thread so that folks can suggest (as you have) improvements to the page-feedback feature.

I'd like to see more values supported, customizable by downstream projects, rather than a special event for "no" answers. But that's just off the top of my head. I'll give this more thought when I have the time.

**Comment by sorenwacker** [comment]
Hi, 

I am seeing the `page_helpful` events in Google analytics, however, how can I distinguish if the user clicked on Yes or No? What I am mostly interested in is where the user clicked No to see where my documentation needs improvement.

**Comment by huanlin** [comment]
Hi,
I have created some documentation sites at the company I work for. The documentation sites are behind a firewall and Google Analytics is not accessible. 
Therefore, I wish the page feedback feature could support some kind of self-hosting options.
